src/test/java/it/pagopa/pn/cucumber/utils/NotificationValue.java

```diff
@@ -32,7 +32,7 @@ public enum NotificationValue {
     INTERNAL_ID("internalId","",false),
     DIGITAL_DOMICILE("digitalDomicile","",false),
     DIGITAL_DOMICILE_TYPE("digitalDomicile_type","PEC",false),
-    DIGITAL_DOMICILE_ADDRESS("digitalDomicile_address","testpagopa3@pnpagopa.postecert.local",false),
+    DIGITAL_DOMICILE_ADDRESS("digitalDomicile_address","notifichedigitali-uat@pec.pagopa.it",false),
```

Comment:
converrebbe che la mail sia parametrica non Hard coded data la necessità di variarla in base all'ambiente

Reply:
fatto